### PR TITLE
Update deprecations

### PR DESCRIPTION
**Pull request by adasiewiczr**
### 🎯 Goal

Update deprecations.

### 🧪 Testing

Run sample apps

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [ ] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works

### CODE REVIEW

**Review by kanat — APPROVED** [review]
LGTM

File: DEPRECATIONS.md [hunk 1]
```diff
@@ -4,32 +4,32 @@ This document lists deprecated constructs in the SDK, with their expected time 
...
+| `RowScope.DefaultComposerInputContent` | 2022.08.02<br/>5.7.0 | 2022.09.06 ⌛ | 2022.10.04 ⌛ | Use `MessageInput` instead. |
+| `LegacyDateFormatter` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `PorterImageView` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `PorterShapeImageView` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `ChatClient::disconnect` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.19 ⌛ | Use `ChatClient.disconnect(Boolean)` instead. |
+| `ChatLogger` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog` instead.|
+| `ChatLogger.Config` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `ChatLoggerConfig` instead.|
+| `ChatLogger::get` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog::getLogger` instead.|
+| `BaseChatModule::logger` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog` instead.|
+| `GlobalState::setUser` | 2022.07.19<br/>5.6.0 | 2022.08.02<br/>5.7.0 | 2022.09.06 ⌛ | Use `ClientMutableState:setUser` instead.|
```
Comment (bychkovdmitry) [inline DEPRECATIONS.md:18]: There is no `GlobalState::setUser` field. Previously we had MutableGlobalState::setUser but it has already been removed and the class itself is internal. I would just remove this line completely.

**Review by bychkovdmitry — APPROVED** [review]
All good, just one minor comment.

File: DEPRECATIONS.md (hunk 1 above)
Comment (adasiewiczr) [reply DEPRECATIONS.md:18]: nice catch. Thank you!

**Pull request merged**